src/resources/js/admin/tickets-commerce-settings.js

```diff
@@ -250,6 +251,53 @@ tribe.tickets.admin.commerceSettings = {};
 			obj.disconnectButton.on( 'click', obj.handleDisconnectClick );
 		}
 
+		$document.on( 'click', obj.selectors.webHookSetUpLink, function( evt ) {
+			evt.preventDefault();
+			const target = $( evt.target );
+
+			const removeIfExists = $document.find( '#tec-tickets-webhook-creation-update' );
+
+			if ( removeIfExists.length ) {
+				removeIfExists.remove();
+			}
+
+			$(
+				`<p id="tec-tickets-webhook-creation-update" class="tooltip description">
+				<span class="dashicons dashicons-update"></span>
+				<span>${ target.attr( 'data-loading-text' ) }</span>
+				</p>`,
+			).insertAfter( target.closest( 'p' ) );
+
+			fetch( target.attr( 'href' ) ).then( function( res ) {
+				return res.json();
+			} ).then( function( res ) {
+				if ( true !== res.success ) {
+					$( '#tec-tickets-webhook-creation-update' )
+						.find( 'span:not(.dashicons)' )
+						.text( res.data.status );
+
+					$( '#tec-tickets-webhook-creation-update' )
+						.find( 'span.dashicons' )
+						.removeClass( 'dashicons-update' )
+						.addClass( 'dashicons-no' );
+					return;
+				}
+
+				$( '#tec-tickets-webhook-creation-update' )
+					.find( 'span:not(.dashicons)' )
+					.text( res.data.status );
+
+				$( '#tec-tickets-webhook-creation-update' )
+					.find( 'span.dashicons' )
+					.removeClass( 'dashicons-update' )
+					.addClass( 'dashicons-yes' );
+
+				setTimeout( function() {
+					window.location.reload();
+				}, 700 );
+			} );
```

Comment:
Why duplicate the code?
```suggestion
			} ).then( function( res ) {
				const $update_ele = $update_ele;
				const $update_dashicon = $update_ele.find( 'span.dashicons' );

				$update_ele
					.find( 'span:not(.dashicons)' )
					.text( res.data.status );

				$update_dashicon.removeClass( 'dashicons-update' );

				if ( true !== res.success ) {
					$update_dashicon.addClass( 'dashicons-no' );
					return;
				}

				$update_dashicon.addClass( 'dashicons-yes' );

				setTimeout( function() {
					window.location.reload();
				}, 700 );
			} );
```

Reply:
Yes that makes sense. Implemented similar to your suggestion, Thank you!